_data/language.yaml

```diff
@@ -23,3 +23,7 @@ array:
   - name: C++
     url: //github.com/opentracing/opentracing-cpp
     icon: ''
+
+  - name: Contributions
+    url: //github.com/opentracing-contrib
```

Comment:
@pavolloffay , after you change the link's position, and link itself. This PR will be ready to be merged. 😊